docs/source/ingest/destination_connectors/s3.rst

```diff
@@ -0,0 +1,68 @@
+S3
+===========
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs locally on your filesystem and upload those local files to an S3 bucket.
+
+First you'll need to install the S3 dependencies as shown here.
+
+.. code:: shell
+
+  pip install "unstructured[s3]"
+
+Run Locally
+-----------
+The upstream connector can be any of the ones supported, but for convenience here, showing a sample command using the
+upstream s3 connector. This will create new files on your local.
+
+.. tabs::
+
+   .. tab:: Shell
+
+      .. code:: shell
+
+        unstructured-ingest \
+            s3 \
```

Comment:
Should use local connector here for the source connector

Reply:
Addressed with [s3 docs pythonic approach and local connector](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/4f65b4965b0278957491c738c2ba08a33b96b3fd)